pkg/http/api.go

```diff
@@ -382,10 +385,27 @@ func (a *api) onGetSecret(c *routing.Context) error {
 		return nil
 	}
 
+	metadata := map[string]string{}
+
+	secretNamespace := string(c.QueryArgs().Peek(secretNamespaceParam))
```

Comment:
Instead of hard-coding specific query params, we should iterate over every query argument, check for those that have a `metadata` prefix and append them to `metadata := map[string]string{}`.

Reply:
Fixed in #a1ffb52